docs/frost-bookends.md

```diff
@@ -0,0 +1,67 @@
+# frost-bookends <br />
+
+ * [API](#api)
+ * [Examples](#examples)
+
+## API
+
+| Attribute   | Type | Value | Description |
+| ----------- | ---- | ----- | ----------- |
+| `content` | `component` | | **required** - component to render as the content (between header and footer) |
```

Comment:
As noted above around the yield case, I think we could make the `content` optional to support blockless/block formats